### DISCUSSION TITLE
Mysql, Redis... not working ?

### DISCUSSION THREAD

**Discussion by metal3d** [nfrastack/container-lemonldap#49] [Q&A]
Hello
Glade to see that someone as configured a Dockerfile with a lot of environement variables, I tried to startup lemonldap with other CONFIG_TYPE than FILE, and none won't to startup.

REDIS => it only want IP address, not hostname

MYSQL => 
```

** [lemonldap] Starting nginx
--
  |  
  | **** [zabbix] Starting Zabbix Agent
  |  
  | ** [lemonldap] Starting LemonLDAP FastCGI Server
  | Lemonldap::NG::Handler::SharedConf : unable to build configuration: Cannot read /etc/lemonldap-ng/lemonldap-ng.ini to get configuration access parameters.
  | Error: configStorage: type is not defined.

```

Have you ever tried them ? and if yes, can you provide a docker-compose example or any environment configuration that let you start it ?

Tnanks a lot

**Comment by tiredofit** [comment]
Hi there, we're currently using redis as a session stroage and config as json (default) along with multiple handlers pulling from the portal via SOAP.  When I built, I did very basic tests to ensure it worked, and then moved onto the next configuration option as it was all purely learning and wanting to make sure that I could figure out the right option for my environment. Can I assume you have already run it as FILE and then converted the configuration to your DBMS format already? As per the LLNG manual you will need to start with FILE and then switch over to a different backend configuration.

Regardless, there may have been something that snuck in, so I will walk through the scripts I built, as something might have happened when I built the Alpine versions wihch were pretty challenging to get right.
